### PR TITLE
[PPM-292]  Remove client link measuruments 11k

### PR DESCRIPTION
**Pull request by VladyslavTupikin**
Need to remove next messages and all related code:

```
ACTION_CONTROL_CLIENT_LINK_MEASUREMENT_11K_REQUEST
ACTION_CONTROL_CLIENT_LINK_MEASUREMENTS_11K_RESPONSE
````

### CODE REVIEW

File: agent/src/beerocks/fronthaul_manager/monitor/monitor_thread.cpp [hunk 1]
```diff
@@ -1760,46 +1760,6 @@ bool monitor_thread::hal_event_handler(bwl::base_wlan_hal::hal_event_ptr_t event
 
     } break;
 
-    case Event::RRM_Link_Measurement_Response: {
```
Comment (adam1985d) [inline agent/src/beerocks/fronthaul_manager/monitor/monitor_thread.cpp:1779]: @arnout is this message really deprecated or should it remain and its information is translated to a non-VS message?

File: common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml [hunk 2]
```diff
@@ -271,8 +271,7 @@ eActionOp_MONITOR:
   ACTION_MONITOR_CLIENT_CHANNEL_LOAD_11K_RESPONSE: 32
   ACTION_MONITOR_CLIENT_STATISTICS_11K_REQUEST: 33
   ACTION_MONITOR_CLIENT_STATISTICS_11K_RESPONSE: 34
-  ACTION_MONITOR_CLIENT_LINK_MEASUREMENT_11K_REQUEST: 35
-  ACTION_MONITOR_CLIENT_LINK_MEASUREMENTS_11K_RESPONSE: 36
+
```
Comment (adam1985d) [inline common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml]: please remove the line space

File: common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_monitor.yaml [hunk 3]
```diff
@@ -159,14 +159,6 @@ cACTION_MONITOR_CLIENT_STATISTICS_11K_RESPONSE:
   _type: class
   params: sStatisticsResponse11k
```
Comment (adam1985d) [inline common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_monitor.yaml:161]: this commit and the previous one (of the auto-generated code) should be in the same commit IMO

File: common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml [hunk 4]
```diff
@@ -330,8 +330,8 @@ eActionOp_CLI:
   ACTION_CLI_CLIENT_DISALLOW_REQUEST: 81
   ACTION_CLI_CLIENT_DISCONNECT_REQUEST: 82
   ACTION_CLI_CLIENT_BSS_STEER_REQUEST: 83
-  ACTION_CLI_CLIENT_LINK_MEASUREMENT_11K_REQUEST: 84
 
+  ACTION_CLI_CLIENT_CHANNEL_LOAD_11K_REQUEST: 85
```
Comment (adam1985d) [inline common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml]: this commit message is about removal - not addition of new CMDUs

File: common/beerocks/tlvf/AutoGenerated/include/beerocks/tlvf/beerocks_message_cli.h [hunk 5]
```diff
@@ -458,18 +458,19 @@ class cACTION_CLI_CLIENT_BSS_STEER_REQUEST : public BaseClass
...
 };
 
-class cACTION_CLI_CLIENT_LINK_MEASUREMENT_11K_REQUEST : public BaseClass
+class cACTION_CLI_CLIENT_CHANNEL_LOAD_11K_REQUEST : public BaseClass
 {
     public:
-        cACTION_CLI_CLIENT_LINK_MEASUREMENT_11K_REQUEST(uint8_t* buff, size_t buff_len, bool parse = false);
-        explicit cACTION_CLI_CLIENT_LINK_MEASUREMENT_11K_REQUEST(std::shared_ptr<BaseClass> base, bool parse = false);
-        ~cACTION_CLI_CLIENT_LINK_MEASUREMENT_11K_REQUEST();
+        cACTION_CLI_CLIENT_CHANNEL_LOAD_11K_REQUEST(uint8_t* buff, size_t buff_len, bool parse = false);
```
Comment (adam1985d) [inline common/beerocks/tlvf/AutoGenerated/include/beerocks/tlvf/beerocks_message_cli.h]: again - please provide the tlvf and the autogenerated files in the same commit - no added value in splitting them up

File: common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml [hunk 6]
```diff
@@ -87,8 +87,7 @@ eActionOp_CONTROL:
 
   ACTION_CONTROL_CLIENT_STATISTICS_11K_REQUEST: 121
   ACTION_CONTROL_CLIENT_STATISTICS_11K_RESPONSE: 122
-  ACTION_CONTROL_CLIENT_LINK_MEASUREMENT_11K_REQUEST: 123
-  ACTION_CONTROL_CLIENT_LINK_MEASUREMENTS_11K_RESPONSE: 124
```
Comment (adam1985d) [inline common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml:89]: please provide the tlvf changes and autogenerated files in the same commit.

and please remove the added line space

**Review by adam1985d — CHANGES_REQUESTED** [review]
the only reason I am not approving is that one of the commits adds a CMDU (either by mistake or by purpose) but the commit message says nothing about it.
General suggestions:
1. replace in all commit messages the `Need to remove the..." with "Removed the.."
2. tlvf changes and autogenerated files should be on the same commit IMO
all are nitpicks.

the only real issue: fix the commit that adds new CMDU and trigger re-review

File: common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml (hunk 2 above)
Comment (VladyslavTupikin) [reply common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml]: fixed

File: common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_monitor.yaml (hunk 3 above)
Comment (VladyslavTupikin) [reply common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_monitor.yaml:161]: accepted

File: common/beerocks/tlvf/AutoGenerated/include/beerocks/tlvf/beerocks_message_cli.h (hunk 5 above)
Comment (VladyslavTupikin) [reply common/beerocks/tlvf/AutoGenerated/include/beerocks/tlvf/beerocks_message_cli.h]: fixed

File: common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml (hunk 6 above)
Comment (VladyslavTupikin) [reply common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml:89]: fixed

File: common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml (hunk 4 above)
Comment (VladyslavTupikin) [reply common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml]: fixed

**Review by mariomaz — APPROVED** [review]
LGTM

File: common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml (hunk 2 above)
Comment (adam1985d) [reply common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml:272]: the line space is still here
(nitpick)

Comment (adam1985d) [reply common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_action.yaml]: the line space is still here
(nitpick)

**Review by adam1985d — APPROVED** [review]
I added (or actually unresolved) one comment, but it is a nitpick so approving

**Pull request merged**